src/core/ext/xds/xds_route_config.cc

```diff
@@ -80,10 +80,9 @@
 
 namespace grpc_core {
 
-// TODO(donnadionne): Remove once RLS is no longer experimental
```

Comment:
This TODO needs to stay, although you can change the assignee.  We still want to get rid of this env var guard before too long.

Reply:
SG, I suspect this env var will be useful at least for upcoming weeks while we iron out possible RLS issues beyond the client.

Updated the comment to remove by 1.58